src/sentry/migrations/0182_update_user_misery_on_saved_queries.py

```diff
@@ -0,0 +1,58 @@
+# Generated by Django 1.11.29 on 2021-03-26 14:43
+import re
+
+from django.db import migrations
+
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+USER_MISERY_REGEX = r"^user_misery\((\d+)\)"
+
+
+def update_user_misery_column_on_saved_queries(apps, schema_editor):
+    DiscoverSavedQuery = apps.get_model("sentry", "DiscoverSavedQuery")
+
+    for saved_query in RangeQuerySetWrapperWithProgressBar(DiscoverSavedQuery.objects.all()):
+        query = saved_query.query
+        if "fields" not in query:
+            continue
+
+        fields = query["fields"]
+        for i, field in enumerate(fields):
+            match = re.match(USER_MISERY_REGEX, field)
+            if match and "count_unique(user)" not in fields:
+                fields[i] = f"count_miserable(user, {match.group(1)})"
+                saved_query.query["fields"] = fields
+                saved_query.save()
+                break
+
+
+class Migration(migrations.Migration):
+    # This flag is used to mark that a migration shouldn't be automatically run in
+    # production. We set this to True for operations that we think are risky and want
+    # someone from ops to run manually and monitor.
+    # General advice is that if in doubt, mark your migration as `is_dangerous`.
+    # Some things you should always mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that
+    #   they can be monitored. Since data migrations will now hold a transaction open
+    #   this is even more important.
+    # - Adding columns to highly active tables, even ones that are NULL.
+    is_dangerous = False
```

Comment:
The DiscoverSavedQuery table isn't too big and this migration will only update a small number of rows on the table, so shouldn't be risky